pandas/tests/dtypes/test_dtypes.py

```diff
@@ -40,7 +40,12 @@ def test_equality_invalid(self, dtype):
         assert not is_dtype_equal(dtype, np.int64)
 
     def test_numpy_informed(self, dtype):
-        with pytest.raises(TypeError, match="data type not understood"):
+        # npdev 2020-02-02 changed from "data type not understood" to
+        #  "Cannot interpret 'foo' as a data type"
+        msg = "|".join(
+            ["data type not understood", "Cannot interpret '.*' as a data type"]
+        )
```

Comment:
I think we simply shouldn't test the message here, we are literally testing numpy here